cmd/vic-machine/common/id.go

```diff
@@ -30,16 +28,12 @@ func (i *VCHID) IDFlags() []cli.Flag {
 		cli.StringFlag{
 			Name:        "id",
 			Value:       "",
-			Usage:       "The ID of the Virtual Container Host - not supported until vic-machine ls is ready",
+			Usage:       "The ID of the Virtual Container Host",
```

Comment:
Currently is using full name prefixed by VirtualMachine, in ls and inspect command.
Will change both of them.